pkg/kubelet/server/stats/types.go

```diff
@@ -115,3 +123,23 @@ type MemoryStats struct {
 	// Cumulative number of major page faults.
 	MajorPageFaults *int64 `json:"majorPageFaults,omitempty"`
 }
+
+// VolumeStats contains data about Volume filesystem usage.
+type VolumeStats struct {
+	// Embedded FsStats
+	FsStats
+	// Path is the directory path the Volume is mounted to
```

Comment:
I was basing this off of what is exposed in the volume.Volume interface (only the path and not the volume name).  I can change this to the Volume name.  Do you know why name is not part of the Volume interface, but path is?